crates/ruff_linter/src/rules/pycodestyle/snapshots/ruff_linter__rules__pycodestyle__tests__E305_E30.py.snap

```diff
@@ -30,14 +29,13 @@ E30.py:817:1: E305 [*] Expected 2 blank lines after class or function definition
     = help: Add missing blank line(s)
 
 ℹ Safe fix
-814 814 |     # comment
-815 815 | 
-816 816 |     # another comment
-    817 |+
-    818 |+
-817 819 | a = 1
-818 820 | # end
-819 821 | 
+811 811 | class Class():
+812 812 |     pass
+813 813 | 
+    814 |+
+814 815 |     # comment
+815 816 | 
+816 817 |     # another comment
```

Comment:
Same as above.

The diagnostic highlights the correct location but the fix seems to be incorrect.

```
/Users/dhruv/playground/ruff/src/E305.py:7:1: E305 [*] Expected 2 blank lines after class or function definition, found (1)
  |
6 |     # another comment
7 | a = 1
  | ^ E305
8 | # end
  |
  = help: Add missing blank line(s)

ℹ Safe fix
1 1 | class Class():
2 2 |     pass
3 3 | 
  4 |+
4 5 |     # comment
5 6 | 
6 7 |     # another comment
```